### PR TITLE
Fix >tarot one error

### PR DESCRIPTION
**Pull request by keli5**
Embed field will now have an invisible character as a name instead of `''`

### CODE REVIEW

File: crimsobot/utils/tarot.py [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import io
-import random
```
Comment (williammck) [inline crimsobot/utils/tarot.py:2]: wait why were these removed

Comment (keli5) [reply crimsobot/utils/tarot.py:2]: Oh, whoops. Was removing an accidental newline, must have selected too far

Comment (keli5) [reply crimsobot/utils/tarot.py]: Should be clean now.
